### PR TITLE
Add annotations

### PR DESCRIPTION
**Pull request by troferoller**

### CODE REVIEW

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 1]
```diff
@@ -87,6 +87,10 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
 
     @Test
     @Disabled // EXAMPLE
+    @Story("Account registers a domain manualy")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: If the test is disabled, it doesn't need to be marked up. This is probably just an example.

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 2]
```diff
@@ -956,6 +968,10 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
     }
 
     @Test
+    @Disabled
+    @Story("Account queries an assets with response given fetch size")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: Let's don't touch disabled I think

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 3]
```diff
@@ -1108,7 +1124,9 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
             RubbishToTestMultipleGenesis::class,
         ],
     )
-    @Feature("Configurations")
+    @Feature("Accounts")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: I don't agree. This test checks the correctness of loading iroha with different genesis. In fact it is running iroha with different starting configurations. I think it is possible to return @Feature(“Configurations”).

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 4]
```diff
@@ -1108,7 +1124,9 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
...
+    @Story("Account send multiple instructions")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]:
```suggestion
    @Story("Initiator starts network with multiple genesis")
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 5]
```diff
@@ -1108,7 +1124,9 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
...
+    @SdkTestId("account_send_multiple_instructions")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]:
```suggestion
    @SdkTestId("initiator_start_network_with_multiple_genesis")
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 6]
```diff
@@ -1212,6 +1230,10 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
     }
 
     @Test
+    @Permission("no_permission_required")
+    @Feature("Query")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]:
```suggestion
    @Feature("Configurations")
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 7]
```diff
@@ -1212,6 +1230,10 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
...
+    @Story("Query find all accounts in genesis and apply it")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]:
```suggestion
    @Story("Initiator starts network with fat genesis")
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 8]
```diff
@@ -1212,6 +1230,10 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
...
+    @SdkTestId("query_all_accounts_in_genesis")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]:
```suggestion
    @SdkTestId("initiator_start_network_with_fat_genesis")
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt [hunk 9]
```diff
@@ -125,6 +125,9 @@ class PeerTest : IrohaTest<AdminIroha2Client>() {
     }
 
     @Test
+    @Disabled
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt]: why disabled?

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt [hunk 10]
```diff
@@ -125,6 +125,9 @@ class PeerTest : IrohaTest<AdminIroha2Client>() {
...
+    @Story("")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt]: Let's just don't touch PeerTest yet.

**Review by alexstroke1 — CHANGES_REQUESTED** [review]

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 2 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: Set as disabled cuse of dont understanding for any story to correlate

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 11]
```diff
@@ -114,6 +114,10 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
 
     @Test
     @Disabled // EXAMPLE
+    @Story("Account registers a domain manual via docker-compose")
+    @Feature("Accounts")
+    @Permission("no_permission_required")
+    @SdkTestId("account_registers_domain_manual_via_docker_compose")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: I think let's not mark up tests that are disabled and are examples.

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 12]
```diff
@@ -956,6 +964,9 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
     }
 
     @Test
+    @Story("Account queries an assets with response given fetch size")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]:
```suggestion
    @Story("Account queries an assets with custom fetch size")
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 13]
```diff
@@ -956,6 +964,9 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
...
+    @Permission("no_permission_required")
+    @SdkTestId("query_an_assets_with_respond_given_fetch_size")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]:
```suggestion
    @SdkTestId("query_an_assets_with_custom_fetch_size")
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt [hunk 14]
```diff
@@ -125,6 +125,9 @@ class PeerTest : IrohaTest<AdminIroha2Client>() {
     }
 
     @Test
+    @Story("Round-robin load balancing")
```
Comment (alexstroke1) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt]: Let's don't touch PeerTest.kt yet.

**Review by alexstroke1 — CHANGES_REQUESTED** [review]

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 1 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 3 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 4 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 5 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 7 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt (hunk 9 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt (hunk 10 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 11 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: resolved by deleting annotations

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 13 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: resolved

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt (hunk 14 above)
Comment (troferoller) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/PeerTest.kt]: resolved

**Review by alexstroke1 — APPROVED** [review]

**Review by arndey — APPROVED** [review]

**Pull request merged**